adapter/src/main/java/com/twilio/kudu/sql/schema/KuduSchema.java

```diff
@@ -68,9 +90,14 @@ public KuduSchema(final String connectString, final Map<String, KuduTableMetadat
     // DataLoader (useful only for performance testing)
     this.disableCubeAggregation = Boolean
         .valueOf((String) propertyMap.getOrDefault(DISABLE_CUBE_AGGREGATIONS, "false"));
+    this.disableMaterializedViews = Boolean
+        .valueOf((String) propertyMap.getOrDefault(DISABLE_MATERIALIZED_VIEWS, "false"));
```

Comment:
by default DISABLE_MATERIALIZED_VIEWS = false, does it mean MV is enabled? Should we hace a KJ pr to turn it off as in test?

Reply:
Materialzied views are enabled by default and can be disabled by setting the `DISABLE_MATERIALIZED_VIEWS` property to true.